app/models/form/sales/pages/number_of_others_in_property.rb

```diff
@@ -5,16 +5,18 @@ def initialize(id, hsh, subsection)
     @depends_on = [
       {
         "privacynotice" => 1,
+        "jointpur" => 2,
       },
       {
         "noint" => 1,
+        "jointpur" => 2,
       },
     ]
   end
 
   def questions
     @questions ||= [
-      Form::Sales::Questions::NumberOfOthersInProperty.new(nil, nil, self),
+      Form::Sales::Questions::NumberOfOthersInProperty.new(nil, nil, self, joint_purchase: false),
```

Comment:
can we set joint_purchase dynamically and remove the nede for number_of_others_in_property_joint_purchase.rb? Or is that more effort than it's worth?

Reply:
Yeah, probably, have a look now?